### PR TITLE
[DEMO] db support ph.IV

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
* Configured save-demo to be able to interact with mysql database in terms of kubernetes

### CODE REVIEW

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

File: save-cloud-charts/save-cloud/templates/demo-deployment.yaml [hunk 1]
```diff
@@ -24,20 +24,71 @@ spec:
...
+            - name: database-secret
+              mountPath: {{ .Values.mysql.dbPasswordFile }}
           {{- include "spring-boot.management" .Values.demo | nindent 10 }}
           resources:
             limits:
               memory: 1000M
             requests:
               memory: 800M
+      initContainers:
+        - name: liquibase-runner
```
Comment (nulls) [inline save-cloud-charts/save-cloud/templates/demo-deployment.yaml:44]: git cloner is missed

**Review by nulls — CHANGES_REQUESTED** [review]
git-cloner is missed -- need to download project to fetch liquibase scripts

File: save-cloud-charts/save-cloud/templates/demo-deployment.yaml (hunk 1 above)
Comment (sanyavertolet) [reply save-cloud-charts/save-cloud/templates/demo-deployment.yaml:44]: Oh, got it, will fix asap

Comment (sanyavertolet) [reply save-cloud-charts/save-cloud/templates/demo-deployment.yaml:57]: Done

**Review by nulls — APPROVED** [review]

**Pull request merged**